packages/web-components/examples/codesandbox/components/filter-panel/src/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-web-components/es/components/filter-panel/filter-panel-composite';
```

Comment:
Should this be pulling `index.ts` instead? The ES example looks off:

https://webcomponents-codesandbox.s3-web.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/6443/filter-panel/index.html

![image](https://user-images.githubusercontent.com/1641214/125789373-e3c37341-a73f-40ba-9415-5f100a1f4614.png)